Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -4275,27 +4275,23 @@ void RenderBox::computePositionedLogicalWidthUsing(SizeType widthType, Length lo
 
         const LayoutUnit availableSpace = containerLogicalWidth - (marginLogicalLeftValue + marginLogicalRightValue + bordersPlusPadding);
 
+        auto shrinkToFitLogicalWidth = [this](LayoutUnit availableLogicalWidth, LayoutUnit bordersPlusPadding) -> LayoutUnit {
+            LayoutUnit preferredLogicalWidth = maxPreferredLogicalWidth() - bordersPlusPadding;
+            LayoutUnit preferredMinLogicalWidth = minPreferredLogicalWidth() - bordersPlusPadding;
+            return std::clamp(availableLogicalWidth, preferredMinLogicalWidth, preferredLogicalWidth);
```

Comment:
why not
`return std::clamp(availableLogicalWidth, preferredMinLogicalWidth, preferred`Max`LogicalWidth);`

Reply:
So - I renamed `preferredLogicalWidth` to `preferredMaxLogicalWidth` and then update `return` as well. Can do it as well. :-)

Reply:
Done in https://github.com/WebKit/WebKit/commit/e305b2600d3b504c55223c37217ca2b66d6317af.